extensions/vertx-http/deployment/src/main/resources/dev-templates/io.quarkus.quarkus-vertx-http/config.html

```diff
@@ -146,9 +146,7 @@ <h5>
                         {/if}
                     </td>
                     <td>
-                        {#if item.description.fmtJavadoc && item.description.fmtJavadoc != 'NOT_FOUND'}
-                            {item.description.fmtJavadoc}
-                        {/if}    
+                       {item.description.fmtJavadoc??}
```

Comment:
This is an equivalent of `{item.description.fmtJavadoc.or(null)}` and `null` values produce no output.